.github/PROCESS.md

```diff
@@ -10,13 +10,16 @@ Keep guides short: we don't create rules; we automate.
 
 ## Pull requests
 
-1. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
+1. Send pull requests from forks; do not make personal branches in the main repository.
```

Comment:
Huh, I actually thought I created a fork! Didn't pay enough attention to what GitHub UI suggested to me. 
But that doc line makes total sense!